taskcluster/ci/build-bundle/kind.yml

```diff
@@ -10,8 +10,22 @@ transforms:
     - taskgraph.transforms.job:transforms
     - taskgraph.transforms.task:transforms
 
+kind-dependencies:
+    - toolchain
+
+
 job-defaults:
+    aab-artifact-template:
```

Comment:
I just seized the opportunity to reorder keys by alphabetically order. I didn't know where `fetches` would fit better, so I've come to think ordering alphabetically is an explicit-enough arbitrary rule. Moreover, it makes diffing between projects easier, once all are ordered.